hueniversitypy/theme_ubc.py

```diff
@@ -1,21 +1,75 @@
 def theme_ubc(palette='blue'):
     """ 
     Applies a University of British Columbia theme to all subsequential altair plot objects so they are displayed with the UBC visual identity. 
-    Two palettes based on the UBC visual identity guidelines can be selected: 'blue' and 'white'. 
-	
-    Parameters
-    ----------
-    palette : 'blue' or 'white'
-    	specifies which colour palette defined in the visual identity documentation is used. 
-	
+    See the visual identity at https://brand.ubc.ca/guidelines/downloads/ubc-colour-and-fonts/
+    
     Returns
     -------
     altair plot : altair.vegalite.v4.api.Chart
-    	an altair plot with the UBC visual identity colour theme applied. 
+    an altair plot with the UBC visual identity colour theme applied. 
 
     Example
     ----------
     >>> alt.themes.register('theme_ubc', theme_ubc)
     >>> alt.themes.enable('theme_ubc')
-
     """
+
+    # Typography
+    font = "Arial"
+    labelFont = "Arial" 
+    # Axes
+    axisColor = "#000000"
+    gridColor = "#DEDDDD"
+    # Colors
+    main_palette = ['#002145', '#0055B7', '#00A7E1', '#40B4E5', '#6EC4E8', '#97D4E9']
```

Comment:
```suggestion
    ubc_palette = ['#002145', '#0055B7', '#00A7E1', '#40B4E5', '#6EC4E8', '#97D4E9']
```
Consider renaming the variable to something more specific.